examples/ts-esbuild/package.json

```
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "scripts": {
+    "build": "esbuild --bundle --outdir=build src/*.ts",
+    "test": "uvu -r esbuild-register tests"
+  },
+  "devDependencies": {
+    "esbuild": "^0.12.7",
+    "esbuild-register": "^2.5.0",
+    "uvu": "^0.5.1"
```

Let's pin versions so that there's no room for non-`uvu` dependencies changing/breaking over time. The other examples (should) already be doing the same thing

```suggestion
    "esbuild": "0.12.15",
    "esbuild-register": "2.6.0",
    "uvu": "^0.5.0"
```